PythonRegression/tests/features/steps/stitching_steps.py

```diff
@@ -0,0 +1,82 @@
+from aloe import step
+from util.test_logic import api_test_logic as tests
+from util.transaction_utils import transaction_logic as transactions
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+config = {}
+responses = {}
+
+side_tangle_address = b'SIDE9TANGLE9999999999999999999999999999999999999999999999999999999999999999999999'
+stitching_addresss = b'STITCHING9TRANSACTIONS99999999999999999999999999999999999999999999999999999999999'
+referencing_address = b'REFERENCES9STITCHING9TRANSACTION9999999999999999999999999999999999999999999999999'
+
+@step(r'a stitching transaction is issued on "([^"]*)" with the tag "([^"]*)"')
+def issue_stitching_transaction(step,node,tag):
+    config['nodeId'] = node
+    config['stitchTag'] = tag
+
+    api = tests.prepare_api_call(node)
+    logger.info('Finding Transactions')
+    side_tangle_transactions = api.find_transactions(addresses = [side_tangle_address])
```

Comment:
This call would probably fail since IRI limits the amount of returned Txs: both in amount and in response body size.